src/bin/util.c

```diff
@@ -81,38 +82,84 @@ static const char *basenm(const char *fn)
 	return ptr;
 }
 
-char *cfg_adjust(const char *fn, const char *tmpl, char *buf, size_t len)
+char *cfg_adjust(const char *fn, const char *tmpl, char *buf, size_t len, int sanitize)
```

Comment:
As we attempt to sanitize, do we need to care about symlinks that points outside the scope? I don't really have the context here.

Reply:
Awesome, thanks!  I ended up with a whitelist of absolute paths (/cfg, /media, $HOME), that I check against twice, the second time after having called `realpath()` on the sanitized filename.